procrastinate/worker.py

```diff
@@ -365,10 +407,13 @@ async def _shutdown(self, side_tasks: list[asyncio.Task]):
 
     def _start_side_tasks(self) -> list[asyncio.Task]:
         """Start side tasks such as periodic deferrer and notification listener"""
-        side_tasks = [asyncio.create_task(self.periodic_deferrer(), name="deferrer")]
-        if self.wait and self.listen_notify:
+        side_tasks = [
+            asyncio.create_task(self.periodic_deferrer(), name="deferrer"),
+            asyncio.create_task(self._poll_jobs_to_abort(), name="poll_jobs_to_abort"),
```

Comment:
Given the 2 polling operations are independent, should the intervals be controlled by the same variable ? (can be handled in a different PR)

Reply:
I thought about it too. We can expose 2 different settings.  Happy for it to be a separate PR.